data/currencies.json

```diff
@@ -0,0 +1,34 @@
+{
+  "USD": "USD", "DOLLAR": "USD", "BUCK": "USD", "US DOLLAR": "USD",
```

Comment:
I wrote it this way so that it would always require constant time to check for a translation. I could implement it in the following format:
`{
    "USD": ["DOLLAR", "BUCK", "US DOLLAR"]
}`

This would make it more concise, but it would also slow it down. The slowdown would be negligible in times with little usage, but if Jarvis ever came under heavy load it may be nice to have faster translation.

What do you think?